gi-stubs/repository/Adw.pyi

```diff
@@ -0,0 +1,946 @@
+    def get_developers(*args, **kwargs): ...
+    def get_documenters(*args, **kwargs): ...
+    def get_issue_url(*args, **kwargs): ...
+    def get_license(*args, **kwargs): ...
+    def get_license_type(*args, **kwargs): ...
+    def get_release_notes(*args, **kwargs): ...
+    def get_release_notes_version(*args, **kwargs): ...
+    def get_support_url(*args, **kwargs): ...
+    def get_translator_credits(*args, **kwargs): ...
+    def get_version(*args, **kwargs): ...
+    def get_website(*args, **kwargs): ...
+    def set_application_icon(*args, **kwargs): ...
+    def set_application_name(*args, **kwargs): ...
+    def set_artists(*args, **kwargs): ...
+    def set_comments(*args, **kwargs): ...
+    def set_copyright(*args, **kwargs): ...
+    def set_debug_info(*args, **kwargs): ...
+    def set_debug_info_filename(*args, **kwargs): ...
+    def set_designers(*args, **kwargs): ...
+    def set_developer_name(*args, **kwargs): ...
+    def set_developers(*args, **kwargs): ...
+    def set_documenters(*args, **kwargs): ...
+    def set_issue_url(*args, **kwargs): ...
+    def set_license(*args, **kwargs): ...
+    def set_license_type(*args, **kwargs): ...
+    def set_release_notes(*args, **kwargs): ...
+    def set_release_notes_version(*args, **kwargs): ...
+    def set_support_url(*args, **kwargs): ...
+    def set_translator_credits(*args, **kwargs): ...
+    def set_version(*args, **kwargs): ...
+    def set_website(*args, **kwargs): ...
+    
+
+class ActionRow:
+    def add_prefix(self, widget: Gtk.Widget) -> None: ...
+    def add_suffix(self, widget: Gtk.Widget) -> None: ...
+    def get_activatable_widget(self) -> Gtk.Widget | None: ...
+    def get_icon_name(self) -> str | None: ...
+    def get_subtitle(self) -> str | None: ...
+    def get_subtitle_lines(self) -> int: ...
+    def get_title_lines(self) -> int: ...
+    def remove(self, widget: Gtk.Widget) -> None: ...
+    def set_activatable_widget(self, widget: Gtk.Widget | None) -> None: ...
+    def set_icon_name(self, icon_name: str | None) -> None: ...
+    def set_subtitle(self, subtitle: str) -> None: ...
+    def set_subtitle_lines(self, subtitle_lines: int) -> None: ...
+    def set_title_lines(self, title_lines: int) -> None: ...
+    
+
+class Animation:
+    parent_instance: GObject.Object = ...
+    
+    def get_state(self) -> AnimationState: ...
+    def get_target(self) -> AnimationTarget: ...
+    def get_value(self) -> float: ...
+    def get_widget() -> Gtk.Widget: ...
+    def pause(self) -> None: ...
+    def play(self) -> None: ...
+    def reset(self) -> None: ...
+    def resume(self) -> None: ...
+    def set_target(*args, **kwargs): ...
+    def skip(self) -> None: ...
+    
+
+class AnimationTarget: ...
+
+class Application:
+    def get_style_manager(self) -> StyleManager: ...
+    
+
+class ApplicationWindow:
+    def get_content(self) -> Gtk.Widget: ...
+    def set_content(self, widget: Gtk.Widget | None) -> None: ...
```

Comment:
the union notation was added in python 3.10 i believe, we currently only require 3.7.

Can you use Union[] instead?

Reply:
Ah, I'll start working on that.

Reply:
The union notation should all be replaced by Union[] now.